at-chuongphu/excercise4.rb

```diff
@@ -0,0 +1,20 @@
+# ex1
+array = 'asianstech'.split(' ')
+array.each do |x|
+  puts array.index(x) if array.count(x) == 1
+end
+# # ex2
+def sumarr(a)
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/2c8e56bfd68c49140235e522ada9011081c7c259/at-chuongphu/excercise4.rb#L7"></span>Naming/UncommunicativeMethodParamName: Method parameter must be at least 3 characters long.</td>
    </tr>
  </tbody>
</table>